### PR TITLE
Overwriting service location

### LINKED ISSUE #14: Overwriting the service location with the help of service attributes
The service location for web apps should be able to be changed dynamically. For this purpose, the 'location' attribute is to be read and the location of a service is to be overwritten. 
If there is no location attribute, the name of the services will serve as the location as before.

### PR DESCRIPTION
**Pull request by CloudFlo2312**
Resolves #14

### CODE REVIEW

File: confd/confd_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package confd_test
```
Comment (Subaro) [inline confd/confd_test.go:1]: Hi, there. There are multiple possibilities in golang when testing the same method for different cases. It is recommended to group all test cases in the method under test. Take a look at some of the test cases in the cesapp for better understanding: https://github.com/cloudogu/cesapp/blob/develop/containers/containerProviderRegistry_test.go

We should group the test cases by the `GetStringValue`-method and the `GetAttributes`-method.

File: confd/service/service_test.go [hunk 2]
```diff
@@ -61,3 +67,42 @@ func TestCreateServiceWithNonStringNameKey(t *testing.T) {
 	service := createService(raw)
 	require.Nil(t, service)
 }
+
```
Comment (Subaro) [inline confd/service/service_test.go:70]: We should create a `TestCreateService` with multiple `t.Run("Create service ...)`

**Review by jsprey — CHANGES_REQUESTED** [review]

File: confd/service/service_test.go (hunk 2 above)
Comment (CloudFlo2312) [reply confd/service/service_test.go]: I have created a `TestCreateService` method with multiple `t.Run`s

File: confd/confd_test.go (hunk 1 above)
Comment (CloudFlo2312) [reply confd/confd_test.go:1]: I have grouped the test cases by the `GetStringValue`-method and `GetAttributes`-method.

**Review by jsprey — APPROVED** [review]

**Pull request merged**